modules/administration-guide/partials/proc_configuring-bitbucket-server-oauth1.adoc

```diff
@@ -0,0 +1,117 @@
+= Configuring Bitbucket server OAuth1
+
+OAuth1 for Bitbucket server allows for automatic obtaining and renewing link:https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html[Personal access tokens]. These tokens {prod-short} used to resolve devfile in a factory flow or it  xref:end-user-guide:authentication-against-bitbucket-server-with-the-personal-access-token.adoc[can be used in place of passwords for Git over HTTPS].
+
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* Bitbucket server installed and reachable from {prod-short} server.
+
+.Procedure
+
+* Generate RSA key pair as described below or follow this guide: link:https://confluence.atlassian.com/jirakb/how-to-generate-public-key-to-application-link-3rd-party-applications-913214098.html[How to generate public key to application link 3rd party applications]
+
+[subs="+quotes,+attributes"]
+----
+openssl genrsa -out private.pem 2048
+openssl rsa -in private.pem -pubout > public.pub
+openssl pkcs8 -topk8 -inform pem -outform pem -nocrypt -in private.pem -out privatepkcs8.pem
+----
+
+* Generate consumer key and shared secret.
+
+[subs="+quotes,+attributes"]
+----
+openssl rand -base64 24 > bitbucket_server_consumer_key
+openssl rand -base64 24 > bitbucket_shared_secret
+----
+
+*  Configure an link:https://confluence.atlassian.com/adminjiraserver/using-applinks-to-link-to-other-applications-938846918.html[Application Link] in Bitbucket to allow {prod-short} communicate with your enterprise Bitbucket server. To create the Application Link:
+**  In Bitbucket Server, go to **Administration** (select the cog in the top navigation bar)  > **Application Links**.
+
+image::bitbucket/bitbucket_configure_application_links.png[link="../_images/bitbucket/bitbucket_configure_application_links.png"]
+**  Enter the application URL (see Application Link details page) and select **Create new link**.
+**  Select **Continue** on the warning message. This is not a problem.
+**  Complete the form:
+   - Application Name - Enter a name to help you identify this {prod-short} instance.
+   - Application Type - Leave as Generic Application.
+   - Service Provider Name - Enter the same name you used for Application Name.
+   - Consumer Key - Specify a consumer key. That is the content of `bitbucket_server_consumer_key` file.
+   - Shared secret - Specify shared secret. That is the content of `bitbucket_shared_secret` file.
+   - Request Token URL - `{your Bitbucket Server URL}/plugins/servlet/oauth/request-token`.
+   - Access token URL - `{your Bitbucket Server URL}/plugins/servlet/oauth/access-token`.
+   - Authorize URL - `{your Bitbucket Server URL}/plugins/servlet/oauth/access-token`.
+   - Create incoming link - Select this checkbox.
+[NOTE]
+====
+Bitbucket is not going to communicate with {prod-short}. There is no outgoing integration.
+It doesn't really matter what value is set as in`Request Token URL`, `Access token URL`, or `Authorize URL`.
```

Comment:
```suggestion
It doesn't really matter what value is set as in `Request Token URL`, `Access token URL`, or `Authorize URL`.
```
I think it should fix an issue with displaying `
![Screenshot_20210203_105404](https://user-images.githubusercontent.com/5887312/106722097-37ff8500-660e-11eb-8984-35e0e55d1832.png)